src/AssetPack/AssetPack.router.ts

```diff
@@ -134,33 +135,42 @@ export class AssetPackRouter extends Router {
 
     let assetPacks: FullAssetPackAttributes[] = []
 
-    // Get default asset packs
-    if (!ethAddress || ethAddress !== DEFAULT_ETH_ADDRESS) {
-      const defaultAssetPacks = await this.logExecutionTime(
-        this.retrieveDefaultAssetPacks,
+    // Get user asset packs
+    if (ethAddress) {
+      const userAssetPacks = await this.logExecutionTime(
+        () => AssetPack.findByEthAddressWithAssets(ethAddress),
         `Get the user\'s (${ethAddress}) asset packs`,
         tracer
       )
-      assetPacks = [...defaultAssetPacks]
-      this.logger.info(
-        `[${tracer}] Assets pack length after adding the default asset packs: ${assetPacks.length}`
-      )
+
+      // If the user doesn't have any asset packs, return the default asset packs
+      if (userAssetPacks.length === 0) {
+        return this.sendDefaultAssetPacksRaw(res, tracer)
+      }
+
+      assetPacks = [...assetPacks, ...userAssetPacks]
+    } else {
```

Comment:
`assetPacks` is always empty at this stage, right?

Maybe we can use it instead of creating `userAssetPacks` variable.


What do you think of doing something like 

```ts
if (ethAddress) {
assetPacks = await this.logExecutionTime(
        () => AssetPack.findByEthAddressWithAssets(ethAddress),
        `Get the user\'s (${ethAddress}) asset packs`,
        tracer
      )
}
if (assetPacks.length === 0) { 
return this.sendDefaultAssetPacksRaw(res, tracer)
}

...
```

Reply:
Done!